### PR TITLE
chore(doc): add monitoring manual testing doc in testplan

### PR DESCRIPTION
**Pull request by harshshekhar15**
Signed-off-by: Harsh Shekhar <harsh.shekhar@mayadata.io>

This PR intends to do the following:
- Add Monitoring manual tests doc in testplan.

#### Versions:

```
$ Kubernetes version
$ Kubernetes platform
$ kubectl version

```

### CODE REVIEW

**Review by amitbhatt818 — APPROVED** [review]
/lgtm

File: testplan/docs/manual/kubera-metrics.md [hunk 1]
```diff
@@ -1,23 +0,0 @@
----
-id: kubera-metrics
-title: Kubera Manual Testing Metrics
-sidebar_label: Kubera metrics
----
-
-### Below test cases are executed manully for diffrent kind of available metrics
```
Comment (prabhatkumarthakur) [inline testplan/docs/manual/kubera-metrics.md:7]: typo different

File: testplan/docs/manual/kubera-metrics.md [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-
-
-#### To verify volumes details are shown 
-- volume details are shown such as volume name,status,capacity,Application,namespace,replicas and cas type 
-
-#### To verify the report generation functionality 
-- Volume/pool analytics report should get generated 
-
-#### To verify different component status of volume 
-- Deatiled volumes info are shown in graphical way such as IOPS,throughput, latency etc
```
Comment (prabhatkumarthakur) [inline testplan/docs/manual/kubera-metrics.md:17]: typo detailed

**Review by prabhatkumarthakur — APPROVED** [review]
lgtm

**Review by AmitKumarDas — APPROVED** [review]

**Pull request merged**